.rubocop.yml

```diff
@@ -98,6 +98,11 @@ Rails/OutputSafety:
   Exclude:
   - lib/i18n_override.rb
 
+# temporarily disabled until the fix for https://github.com/bbatsov/rubocop/issues/4249
+# is available in a new release
+Rails/RelativeDateConstant:
+  Enabled: false
+
```

Comment:
See https://github.com/18F/identity-idp/pull/1358/commits/8a427ca6f3345ce22c0e87dfd1b837debeac88bc for details.